yamux/src/connection.rs

```diff
@@ -539,13 +542,13 @@ impl<T: AsyncRead + AsyncWrite + Unpin> Active<T> {
             .push_back(Frame::close_stream(id, ack).into());
     }
 
-    fn on_drop_stream(&mut self, id: StreamId) {
-        let stream = self.streams.remove(&id).expect("stream not found");
+    fn on_drop_stream(&mut self, stream_id: StreamId) {
+        let s = self.streams.remove(&id).expect("stream not found");
 
-        log::trace!("{}: removing dropped {}", self.id, stream);
-        let stream_id = stream.id();
+        log::trace!("{}: removing dropped stream {}", self.id, id);
+        let stream_id = id;
```

Comment:
This doesn't compile.

I'd suggest we just name it `stream_id` for the entire function.